mitodl/ocw-studio
# Allow selection of null values for website publish fields in Django admin

#### Pre-Flight checklist



#### What are the relevant tickets?
Closes #1223 

#### What's this PR do?
Adds `blank=True` to more `Website` fields

#### How should this be manually tested?
Go to the Django Admin page for any `Website` and change the selected dropdown option for each of the following fields to "-----" if it isn't already, then save.  You should not get any validation errors.
```
Latest build id live
Latest build id draft
Live publish status
Draft publish status
```
